sdk/eventhub/eventhubs-checkpointstore-blob/package.json

```diff
@@ -61,7 +61,7 @@
   "dependencies": {
     "@azure/event-hubs": "^5.0.0",
     "@azure/logger": "^1.0.0",
-    "@azure/storage-blob": "^12.1.2",
+    "@azure/storage-blob": "^12.2.0-preview.1",
```

Comment:
This is a GA package - are we supposed to depend on previews for our dependencies?

(I know we had this discussion recently with identity and were okay with it (I believe) because it's only a devDependency.)

Reply:
We shouldn't depend on the preview version.  If rush complains about versions we could add the alternative version in `common/config/rush/common-versions.json`

Reply:
Undid it.